### DISCUSSION TITLE
error installing atroPIM and atroDAM with demo data

### DISCUSSION THREAD

**Discussion by clin12345** [atrocore/atrocore#298] [General]
hi...
when im installing with demo data either atroPIM or atroDAM its give me error 500, but if im installing without demo data everything works...is it a bug or im doing something wrong.
i follow your installation instruction
ubuntu 20.04
php 7.4
mysql 5.7

many thanks

**Comment by ich-mag** [comment]
Was you able to solve the issue, as I am running in the same issue
